pyproject.toml

```diff
@@ -0,0 +1,34 @@
+# TODO: Move entire setup.cfg to pyproject.toml
```

Comment:
Ruff does not support reading configuration from `setup.cfg`. Instead of creating `ruff.toml`, I put the config into a minimal `pyproject.toml`, in anticipation of eventually moving setup.cfg to pyproject.toml. LMK if you disagree.

Reply:
Let's go, totally agree! Except for apps, it is safe to move other packages to `pyproject.toml` I think.

Reply:
I think it is a good idea, yes.

Reply:
Okay, I'll open another PR for that, thanks!